tests/ecosystem/upgrade/test_upgrade_ocp.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+from ocs_ci.ocs import ocp
+from ocs_ci.framework import config
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import ManageTest
+from ocs_ci.ocs.cluster import CephCluster, CephHealthMonitor
+
+logger = logging.getLogger(__name__)
+
+
+# TODO: add image type validation (ga to ga , nightly to nightly, newer than current etc.)
+
+
+class TestUpgradeOCP(ManageTest):
+    """
+    1. check cluster health
+    2. check OCP version
+    3. perform OCP upgrade
+    4. check all OCP ClusterOperators
+    5. check OCP version
+    5. monitor cluster health
+    """
+
+    def test_upgrade_ocp(self):
+        """
+
+        Tests OCS stability when upgrading OCP
+
+        """
+
+        ceph_cluster = CephCluster()
+        with CephHealthMonitor(ceph_cluster):
+
+            target_image = config.UPGRADE['ocp_upgrade_version']
```

Comment:
Please use this function to get the version if not specified in the config.
https://github.com/red-hat-storage/ocs-ci/pull/1662/files#diff-68bc5ed905b5a4cefbac287573f4f60fR1856

Something like: 
```python
target_image = config.UPGRADE.get('ocp_upgrade_version')
if not  target_image:
    ocp_channel = config.UPGRADE['ocp_channel']  # This has to be defined in default config e.g. stable-4.3
    ocp_upgrade_version = get_latest_ocp_version(channel=ocp_channel)
```
I see that your target image contain architecture in the name. With this I would go with other value as ocp_arch = 'x86_64'
and then would format this image like 
```
ocp_arch = config.UPGRADE['ocp_arch']
target_image = f"{ocp_upgrade_version}-{ocp_arch}"
```

Reply:
please go with variable name:  ocp_upgrade_version as you will construct target image later on. And as I described this should not be mandatory parameter as if not defined you will go with dynamic approach with get_latest_ocp_version.